keylime/keylime_agent.py

```diff
@@ -723,7 +727,26 @@ def main():
     if keylime_ca == "default":
         keylime_ca = os.path.join(config.WORK_DIR, 'cv_ca', 'cacert.crt')
 
-    server = CloudAgentHTTPServer(serveraddr, Handler, agent_uuid, contact_ip)
+    ima_log_file = None
+    if os.path.exists(config.IMA_ML):
+        ima_log_file = open(config.IMA_ML, 'r', encoding="utf-8")
+
+    tpm_log_file = None
+    if os.path.exists(config.MEASUREDBOOT_ML):
+        tpm_log_file = open(config.MEASUREDBOOT_ML, 'rb')
```

Comment:
The UEFI event log can be loaded into memory instead of reading it every time, because it doesn't change after boot.

Reply:
Yes, reading it only once.